app/templates/components/edit-reminder.hbs

```diff
@@ -1,3 +1,5 @@
 {{#link-to 'reminders.reminder'}}
   <button class='save-reminder-btn' {{action 'saveDirtyDiana' on='click'}}>Save</button>
 {{/link-to}}
+
+<button class='undo-btn' {{action 'rollBackTheRock' on='click'}}>Undo</button>
```

Comment:
@martensonbj We did that in the next PR! Can we request that you look at our next PR? We set a conditional so that the visual cues (there are three of them) only show if the reminder is dirty. The three visual cues are:
-The title of the reminder on the home page is red if dirty
-The save button only appears and is red if dirty
-The undo button only appears and is red if dirty